### ISSUE TITLE
Promise Rejection Issue (Again)

### ISSUE THREAD

**Issue by tobinharris** [weiran/wordpress-gatsby-migrater#9]
Hi

I'm getting this when running, any ideas.

```~/.asdf/installs/nodejs/9.6.1/.npm/lib/node_modules/wordpress-gatsby-migrator/index.js pocketworks.wordpress.2019-03-02.xml pocketworks-gatsby 
(node:10125) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'parent' of undefined
    at Function.exports.update (/Users/tobinharris/.asdf/installs/nodejs/9.6.1/.npm/lib/node_modules/wordpress-gatsby-migrator/node_modules/cheerio/lib/parse.js:69:26)
    at module.exports (/Users/tobinharris/.asdf/installs/nodejs/9.6.1/.npm/lib/node_modules/wordpress-gatsby-migrator/node_modules/cheerio/lib/parse.js:19:11)
    at Function.exports.load (/Users/tobinharris/.asdf/installs/nodejs/9.6.1/.npm/lib/node_modules/wordpress-gatsby-migrator/node_modules/cheerio/lib/static.js:27:14)
    at parseImages (/Users/tobinharris/.asdf/installs/nodejs/9.6.1/.npm/lib/node_modules/wordpress-gatsby-migrator/importer.js:76:34)
    at items.map.item (/Users/tobinharris/.asdf/installs/nodejs/9.6.1/.npm/lib/node_modules/wordpress-gatsby-migrator/importer.js:45:24)
    at Array.map (<anonymous>)
    at Object.importPosts (/Users/tobinharris/.asdf/installs/nodejs/9.6.1/.npm/lib/node_modules/wordpress-gatsby-migrator/importer.js:19:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7)
    at Function.Module.runMain (module.js:705:11)
(node:10125) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:10125) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by EvanAgee** [comment]
I'm getting this as well, any ideas?

**Comment by weiran** [comment]
I only built this against my own WordPress XML export so it's likely because it's something that didn't come up in my own use case. 

If you're willing to send me your export (privately) or an anonymised portion that also causes this crash I'm happy to have a look.

**Comment by weiran** [comment]
This could’ve been resolved by #11, can you retry?

**Comment by weiran** [comment]
Closing, create a new issue if this is still a problem.
